packages/flutter_tools/test/src/fakes.dart

```diff
@@ -222,9 +222,9 @@ class MemoryStdout extends MemoryIOSink implements io.Stdout {
 
 /// A Stdio that collects stdout and supports simulated stdin.
 class FakeStdio extends Stdio {
-  final MemoryStdout _stdout = MemoryStdout();
+  final MemoryStdout _stdout = MemoryStdout()..terminalColumns = 80;
   final MemoryIOSink _stderr = MemoryIOSink();
-  final StreamController<List<int>> _stdin = StreamController<List<int>>();
+  final FakeStdin _stdin = FakeStdin();
```

Comment:
Previously this was failing the cast in https://github.com/christopherfujino/flutter/blob/improve-run-test/packages/flutter_tools/lib/src/base/terminal.dart#L286.